spin.lic

```diff
@@ -26,8 +26,9 @@ class Spin
     }
 
     data = mat_data[args.material]
-
-    ensure_copper_on_hand(data[:cost] * data[:count], get_settings.hometown)
+    settings = get_settings
+    @hometown = settings.hometown
```

Comment:
Do lines 34 and 39 need to be updated to use `@hometown`? Or rather, `hometown` since I don't think it needs to be an instance var?

Reply:
I can review. I left instance variable in where the variable was already that or used in another function.

Reply:
I can review. I left instance variable in where the variable was already that or used in another function.

Reply:
Updated.